portfolio/src/main/java/com/google/sps/data/CommentDatabase.java

```diff
@@ -0,0 +1,53 @@
+package com.google.sps.data;
+import com.google.sps.data.Comment;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import java.util.ArrayList;
+
+/* purpose: to manage the interface of the Datastore database used to store comments */
+public final class CommentDatabase {
+  public static final String COMMENT_FIELD = "Comment";
```

Comment:
Why is this field capitalized? What is a field? Is it a row? Or a table? Or both?

Reply:
Capitalized to be a constant, but I should definitely use a more descriptive name.